pubspec.yaml

```diff
@@ -16,6 +16,7 @@ dependencies:
   firebase_auth: ^3.3.19
   cloud_firestore: ^3.1.17
   provider: ^6.0.3
+  fluttertoast: ^8.0.9
```

Comment:
The problem with this package is that it's not supported on iOS devices.
So we can't use it for sorry.
I think we don't need to show feedback after login or logout, just navigate them to the new page.